sklearn/pipeline.py

```diff
@@ -81,10 +84,11 @@ class Pipeline(_BaseComposition):
 
     Parameters
     ----------
-    steps : list of tuple
-        List of (name, transform) tuples (implementing `fit`/`transform`) that
-        are chained in sequential order. The last transform must be an
-        estimator.
+    steps : list of tuples
+        List of (name of step, transformer implementing `transform`) tuples
+        that are to be chained in sequential order. To be compatible with the
+        scikit-learn API, all steps must define `fit`. All non-last steps must
+        also define `transform`.
```

Comment:
this can also be rephrased. Maybe like:
```suggestion
        List of (name, estimator) tuples that are to be chained in sequential
        order. All steps except the last must implement `fit` and `transform`,
        and the last step doesn't have to implement `transform`. See
        :ref:`Combining Estimators <combining_estimators>` for more details.
```

Reply:
I will suggest is mixture of both ;)